aviatrix/resource_aviatrix_firewall.go

```diff
@@ -55,6 +55,7 @@ func resourceAviatrixFirewall() *schema.Resource {
 			"policy": {
 				Type:        schema.TypeList,
 				Optional:    true,
+				Deprecated:  "Please use the standalone aviatrix_firewall_policy resource instead, and set `manage_firewall_policies` to false.",
```

Comment:
This is not following the format as proposed in all the other resources for deprecated note